templates/partials/personas-row.html

```diff
@@ -0,0 +1,36 @@
+<div class="row mb-2 mx-auto">
+    <div class="col-md-12 ">
+        <div class="row g-0 border rounded overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative info-background">
+            {% if img_location == "left" %}
+                <div class="col-2 col-md-3 d-block d-lg-block">
+                    <img class="w-sm-100 mt-sm-5 mt-lg-0 mx-auto d-block img-fluid {{ img_class|default('') }}" src="{{ img_src }}" width="100%"/>
+                </div>
+            {% endif %}
+            {% if second_img_location == "second_left" %}
```

Comment:
My initial plan for the mobile view was to change the second image into three separate images for the icons and align them below the text.  But this morning I figured it would be easier just to hide the second image, given that the first image is already fairly large.

Changed the second_img_location values to just right and left.